nextmv/nextmv/cloud/application.py

```diff
@@ -3867,70 +3867,48 @@ def __input_set_for_scenario(self, scenario: Scenario, scenario_id: str) -> Inpu
     def __validate_input_dir_path_and_configuration(
         self,
         input_dir_path: Optional[str],
-        configuration: Optional[Union[RunConfiguration, dict[str, Any]]],
+        configuration: Optional[RunConfiguration],
```

Comment:
The `__get_input_type` func assumes it might be a dict, so I think the previous type hinting is correct.